.goreleaser.yaml

```
@@ -288,6 +288,30 @@ notarize:
 snapshot:
   version_template: "{{ incpatch .Version }}-snapshot"
 
+homebrew_casks:
+  - name: dbc
+    alternative_names:
+      - dbc@{{ .Version }}
+      - dbc@{{ .Major }}
+      - dbc@{{ .Major }}.{{ .Minor }}
+    ids:
+      - archives
+    url:
+      template: "https://github.com/columnar-tech/dbc/releases/download{{ .Tag }}/{{ .ArtifactName }}"
+      verified: "github.com/columnar-tech/dbc/"
+      using: ":homebrew_curl"
+      cookies:
+        license: "accept-backup"
+    commit_msg_template: "Brew cask update for {{ .ProjectName }} version {{ .Tag }}"
+    homepage: "https://columnar.tech/dbc"
+    description: "A CLI utility for managing ADBC drivers"
+    repository:
+      owner: "columnar-tech"
+      name: "homebrew-dbc"
```

```suggestion
      name: "homebrew-tap"
```

It's slightly more conventional to call the repo "homebrew-tap". That carries the benefit of us not having to rename the repo if we create more formulae/casks.